app/(inferenceai)/inferenceai/components/footer.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import data from "@/locales/inferenceai-data.json";
```

Comment:
this is wrong. please read https://github.com/nicnocquee/simple-i18n-next#readme on how to use the localization.